mm2src/coins/nft/storage/wasm/wasm_storage.rs

```diff
@@ -607,8 +619,9 @@ impl TableSignature for NftListTable {
 }
 
 #[derive(Clone, Debug, Deserialize, Serialize)]
-pub(crate) struct NftTxHistoryTable {
+pub(crate) struct NftTransferHistoryTable {
     transaction_hash: String,
+    log_index: BeBigUint,
```

Comment:
how big can this index be? if it's not going to get up to `u64` i suggest using `u32` here instead of `BeBigUint`

Reply:
I used `BeBigUint` since this value is gotten from this `log_index` https://github.com/KomodoPlatform/komodo-defi-framework/blob/c121c41d9c05aab0f45e0fb1fbe75ddf8e6b069f/mm2src/coins/nft/nft_structs.rs#L378 but I can make the original `log_index` u32 for sure since theoretically it can't be larger than the maximum value of u32. Thanks for the suggestion, will fix it.

Reply:
Done